src/analyses/variable-sensitivity/value_set_abstract_object.h

```diff
@@ -0,0 +1,228 @@
+/*******************************************************************\
+
+ Module: analyses variable-sensitivity
+
+ Author: diffblue
+
+\*******************************************************************/
+
+/// \file
+/// Value Set Abstract Object
+
+#ifndef CPROVER_ANALYSES_VARIABLE_SENSITIVITY_VALUE_SET_ABSTRACT_OBJECT_H
+#define CPROVER_ANALYSES_VARIABLE_SENSITIVITY_VALUE_SET_ABSTRACT_OBJECT_H
+
+#include <unordered_set>
+
+#include <analyses/variable-sensitivity/abstract_value.h>
+
+class value_set_abstract_objectt : public abstract_valuet
+{
+public:
+  using abstract_object_sett = std::unordered_set<
+    abstract_object_pointert,
```

Comment:
I believe this is incorrect? This should just be `exprt`.

Reply:
Yes, I believe so too.

Reply:
I don't know the ins and outs, but that would be a major change, because `abstract_object_pointert` doesn't inherit from `exprt`